CMake/External_log4cplus.cmake

```diff
@@ -11,18 +11,18 @@ ExternalProject_Add(log4cplus
       ${COMMON_CMAKE_ARGS}
       -DCMAKE_CXX_COMPILER:FILEPATH=${CMAKE_CXX_COMPILER}
       -DCMAKE_C_COMPILER:FILEPATH=${CMAKE_C_COMPILER}
+      -DUNICODE:BOOL=OFF
     )
 
 fletch_external_project_force_install(PACKAGE log4cplus)
 
-set(Log4cplus_ROOT ${fletch_BUILD_INSTALL_PREFIX} CACHE STRING "")
-set(Log4cplus_DIR "${Log4cplus_ROOT}/lib/cmake/log4cplus" CACHE PATH "" FORCE)
```

Comment:
hmm
I changed the L to an l
it's correct on my branch... maybe a merge error?

https://github.com/aaron-bray/fletch/blob/release/CMake/External_log4cplus.cmake